### DISCUSSION TITLE
Defining requested permissions for Windows App

### DISCUSSION THREAD

**Discussion by faustbrian** [NativePHP/laravel#448] [Q&A] [answered]
We're about to wrap up an application for production release and everything works fine on Mac, Linux and Windows. Great job!

The only thing we noticed is that the Windows executable asks for permissions to access Location services. Our application doesn't require access to anything special other than printers. I couldn't find anything in the documentation as to how to modify which permissions are requested.

Is there currently a way to change what permissions are being requested when the application launches the first time?

**Comment by simonhamp** [comment]
I haven't seen this myself, so I'm not sure. To my knowledge, Electron doesn't do anything specific here that would trigger this and we certainly don't do anything that I'd expect to trigger this from within NativePHP. So if your app isn't explicitly asking for this, then I suspect the issue lies elsewhere.

**Reply by simonhamp** [reply, marked as answer]
A quick search brought up this very recent, seemingly-related thread in [the Windows forums](https://answers.microsoft.com/en-us/windows/forum/all/let-windows-and-apps-access-your-location-message/76735c77-73b8-4385-9742-982243f67488?page=1), which suggests to me that this might be on OS-level issue.

**Comment by simonhamp** [comment]
I'd love to help you make sure your release is sound before you push it out. Feel free to email me ([github@simonhamp.me](mailto:github@simonhamp.me)) if this is something you'd like to work through together
